package.json

```diff
@@ -12,33 +12,33 @@
     "@gridsome/plugin-sitemap": "^0.4.0",
     "@gridsome/remark-prismjs": "^0.5.0",
     "@gridsome/source-filesystem": "^0.6.2",
-    "@gridsome/transformer-remark": "^0.6.3",
-    "@mdi/font": "^5.8.55",
-    "axios": "^0.21.0",
-    "axios-auth-refresh": "^3.0.0",
+    "@gridsome/transformer-remark": "^0.6.4",
+    "@mdi/font": "^5.9.55",
+    "axios": "^0.21.4",
+    "axios-auth-refresh": "^3.2.2",
     "buefy": "^0.9.17",
     "bulma": "^0.9.3",
     "bulma-divider": "^0.2.0",
-    "bulma-timeline": "^3.0.4",
-    "gridsome": "^0.7.22",
-    "gridsome-plugin-flexsearch": "^1.0.5",
-    "keycloak-js": "^10.0.1",
+    "bulma-timeline": "^3.0.5",
+    "gridsome": "^0.7.23",
+    "gridsome-plugin-flexsearch": "^1.1.0",
+    "keycloak-js": "^11.0.3",
```

Comment:
I see that `keycloack-js` is the only package with a major upgrade (10.x.x -> 11.x.x). Did you test these changes? And why not update to the most recent (17.x.x)? Not sure if [this vulnerability](https://github.com/keycloak/keycloak/security/advisories/GHSA-83x4-9cwr-5487) is relevant but it affects <15.1.1.

Reply:
From my understanding, Keycloak updates all of their packages together: https://www.keycloak.org/docs/latest/release_notes/#keycloak-11-0-0. So I just updated it to the same version that we currently run on our back end: https://github.com/ocf/kubernetes/blob/6fbc98856aefccc078ff4009a8347470c8b9e85d/ocfkube/apps/versions.toml#L42

Reply:
We could probably upgrade to newer versions without any issues, but I would prefer to upgrade our backend version before that

Reply:
Gotcha, sounds good!